GradeServer.c

```diff
@@ -1,146 +1,216 @@
-#include <GradeServer.h>
-#include <ServerThread.h>
+#include <stdio.h>
```

Comment:
All the includes were in the header file, why add it here?

Reply:
Because I changed all the functions, we basically need a new header file